src/common/log/log.ts

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+/**
+ * Logging utility class.
+ */
+
+import {CommandStatus} from "../commandExecutor";
+import {LogHelper, LogLevel} from "./logHelper";
+import {ILogger, OutputChannelLogger, StreamLogger, ConsoleLogger} from "./loggers";
+import {OutputChannel} from "vscode";
```

Comment:
As long as `OutputChannel` is only ever used in a type context, and the emitted javascript has no cause to reference   it, then it should be fine. But that can be a fragile balance.